### PR TITLE
Flesch score

### PR DESCRIPTION
**Pull request by maryesima**
adding method to calculate the Flesch readability score of a document

### CODE REVIEW

File: src/document/Document.java [hunk 1]
```diff
@@ -127,14 +145,18 @@ public String getText()
...
 	{
 	    // TODO: You will play with this method in week 1, and 
 		// then implement it in week 2
-	    return 0.0;
+		int numWords = getNumWords();
+		int numSentences = getNumSentences();
+		int numSyllables = getNumSyllables();
+		double numWordsDivNumSentences = numSentences == 0 ? 0 : ((double)numWords)/numSentences;
+		double numSyllablesDivNumWords = numWords == 0 ? 0 : ((double)numSyllables)/numWords;
+	    return 206.835 - 1.015 * numWordsDivNumSentences - 84.6 * numSyllablesDivNumWords;
```
Comment (etwei-linkedin) [inline src/document/Document.java:160]: TIL this formula haha. Thanks!

File: src/document/Document.java [hunk 2]
```diff
@@ -127,14 +145,18 @@ public String getText()
...
 	public double getFleschScore()
 	{
 	    // TODO: You will play with this method in week 1, and 
 		// then implement it in week 2
-	    return 0.0;
+		int numWords = getNumWords();
+		int numSentences = getNumSentences();
+		int numSyllables = getNumSyllables();
+		double numWordsDivNumSentences = numSentences == 0 ? 0 : ((double)numWords)/numSentences;
+		double numSyllablesDivNumWords = numWords == 0 ? 0 : ((double)numSyllables)/numWords;
```
Comment (etwei-linkedin) [inline src/document/Document.java:159]: Code style: `((double) numWords) / numSentences`

**Comment by etwei-linkedin** [comment]
If you don't want to spend time on code formatter, recommend just downloading the LI Intellij.

File: src/document/Document.java (hunk 1 above)
Comment (maryesima) [reply src/document/Document.java:160]: haha nice!

File: src/document/Document.java (hunk 2 above)
Comment (maryesima) [reply src/document/Document.java:159]: going to update this PR with the correct portion of the file, but noted on making this update as well!

**Pull request closed without merging**